server/src/handlers/http/modal/ingest_server.rs

```diff
@@ -480,7 +373,12 @@ impl IngestServer {
 
         tokio::spawn(airplane::server());
 
-        let app = self.start(prometheus, CONFIG.parseable.openid.clone());
+        // set the ingestor metadata
+        self.set_ingestor_metadata().await?;
+
+        // TODO: figure out why we don't need this
```

Comment:
Why is it that only ingest doesn't care about OpenId?

Reply:
Also why is it that it is the only server that sets it's metadata?

Reply:
Ingest is almost always done via a software like an agent, or Kafka or an application and so on. OAuth based authentication will be a challenge for the workflow